### PR TITLE
Create types-of-loans.md

### PR DESCRIPTION
**Pull request by JamelleWT**
creating the page and moving loan types and eligibility to this page.

### CODE REVIEW

File: pages/loan-basics/types-of-loans.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+sidenav: savings
+
+scripts:
+
+permalink: /loan-basics/types-of-loans/
+
+---
+
+## How a TSP loan works
```
Comment (line47) [inline pages/loan-basics/types-of-loans.md]: Change to header level 1 `#`

File: pages/loan-basics/types-of-loans.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+When you take a loan, you borrow from your TSP account. The amount of your loan cannot exceed the amount of your own contributions and earnings from those contributions. So, if you work under the Federal Employees' Retirement System (FERS), you cannot borrow from any agency contributions or earnings from those contributions.
+ 
+If you meet the loan eligibility rules and your loan request is approved, the loan amount is removed from your TSP account. You must repay your loan with interest. Repayments for the loan and interest are generally made through payroll deductions. Your repayments restore the amount of your loan, plus interest, to your account.
+
+For additional information, visit [Repaying your loan](repaying-your-loan.md).
+
+## Loan types and terms
+
+#### General purpose
```
Comment (line47) [inline pages/loan-basics/types-of-loans.md]: change to header level 3 `###`

File: pages/loan-basics/types-of-loans.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+For additional information, visit [Repaying your loan](repaying-your-loan.md).
+
+## Loan types and terms
+
+#### General purpose 
++ May be used for any purpose
++ Requires no documentation
++ Has a repayment term of 1 to 5 years
+#### Residential
```
Comment (line47) [inline pages/loan-basics/types-of-loans.md]: change to header level 3 `###`

File: pages/loan-basics/types-of-loans.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
++ Requires documentation
++ Has a repayment term of 1 to 15 years
+
+For details, visit Residential Loan Documentation.
+
+![check](tsp-redesign/assets/img/alert-circle.svg "circle alert") Your loan payments must start within 60 days of your loan being disbursed.
+
+## Loan eligibility
+
+#### General eligibility rules
```
Comment (line47) [inline pages/loan-basics/types-of-loans.md]: change to header level 3 `###`

File: pages/loan-basics/types-of-loans.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+**To be eligible for a loan, you:**
++ must be an active federal employee or a member of the uniformed services.
++ must be in pay status because repayments are set up as payroll deductions.
++ can only have one outstanding general purpose loan and one outstanding residential loan from any one TSP account at a time.
++ must have at least $1,000 of your own contributions and earnings in your account (agency contributions and earnings cannot be borrowed).
++ must not have repaid a loan of the same type in full within the past 60 days. (If you have both a civilian account and a uniformed services account, the 60-day waiting period applies separately to each account.)
++ must not have had a taxable distribution of a loan within the past 12 months unless it was the result of your separation from federal service.
++ must not have a court order against your account.
+
+#### Residential loan eligibility rules
```
Comment (line47) [inline pages/loan-basics/types-of-loans.md]: change to header level 3 `###`

**Comment by KishaT** [comment]
@JamelleWT This is good to go now.

**Pull request merged**